packages/common/src/access/can-edit-site.ts

```diff
@@ -0,0 +1,19 @@
+import { IItem, IUser } from "@esri/arcgis-rest-types";
+import { getProp } from "../objects";
+import { hasBasePriv } from "./has-base-priv";
+import { canEditItem } from "./can-edit-item";
+
+/**
+ *
```

Comment:
Please add description. And particularly that this has special logic for "default Hub Home". In the near future we'll be adding "hub home" capability and it should be called out that this method will likely need to be updated.